test/Libraries/DynamoUtilitiesTests/DynamoUtilitiesTests.csproj

```diff
@@ -21,10 +21,6 @@
       <Name>DynamoUtilities</Name>
       <Private>False</Private>
     </ProjectReference>
-    <ProjectReference Include="..\..\..\src\Tools\DynamoInstallDetective\DynamoInstallDetective.csproj">
```

Comment:
I guess for these project references I would also need to make them conditional based on the sln being built - more complicated than I was hoping.